pkgs/development/ocaml-modules/opti/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, fetchurl, buildDunePackage }:
+
+buildDunePackage rec {
+  pname = "opti";
+  version = "1.0.3";
+
+  src = fetchurl {
+    url = "https://github.com/magnusjonsson/opti/releases/download/${version}/opti-${version}.tbz";
+    sha256 = "ed9ba56dc06e9d2b1bf097964cc65ea37db787d4f239c13d0dd74693f5b50a1e";
+  };
+
+  meta = {
```

Comment:
if you would write this as `meta = with stdenv.lib; {` then you can remove the `stdenv.lib` from line 14 and 15.